vtkext/private/module/vtk.module

```diff
@@ -1,26 +1,38 @@
 NAME
-  f3d::VTKExtensionsRendering
+  f3d::vtkextPrivate
 DESCRIPTION
   A VTK containing class related to rendering
 DEPENDS
+  VTK::CommonCore
```

Comment:
Since it is a private module, do we need any public `DEPENDS`?

Reply:
I'm not sure what you mean, but this module depends publicly from VTK::CommonCore

The fact that we use this module only in a private way has no impact on the module itself, just the way it is installed.

Reply:
Yes but since it's private, it won't be installed (other than the library itself) so I think everything should be in `PRIVATE_DEPENDS`

Reply:
incorrect